pages/results.js

```diff
@@ -52,21 +53,24 @@ export default function Results({ data }) {
         totalEmissions={data["total-co2-emissions-tonnes"]}
         totalTripCount={data["total-trip-count"]}
       />
+      <ResultContentSection isShaded={true}>
+        <TopThree
+          topThree={data["TopThreeData"]}
+          totalDistance={data["total-distance"]}
+          totalTripCount={data["total-trip-count"]}
+          totalEmissions={data["total-co2-emissions-tonnes"]}
+        />
 
-      <TopThree
-        topThree={data["TopThreeData"]}
-        totalDistance={data["total-distance"]}
-        totalTripCount={data["total-trip-count"]}
-        totalEmissions={data["total-co2-emissions-tonnes"]}
-      />
-
-      <TripCountAndTravelMethods dataAboutTrips={data["dataAboutTrips"]} />
-      <WorkArrangement workMode={data["work-mode"]} />
-      <CommuteDays data={data} />
-      <CommuteDistanceDistribution data={data} />
-      <DistanceTravelledMode
-        data={data["distance-travelled-by-mode"]}
-      />        
+        <TripCountAndTravelMethods dataAboutTrips={data["dataAboutTrips"]} />
+      </ResultContentSection>
+      <ResultContentSection isShaded={false}>
+        <WorkArrangement workMode={data["work-mode"]} />
+        <CommuteDays data={data} />
+      </ResultContentSection>
```

Comment:
I was going for uniformity, so that anyone else coming in will know that each light or dark section sits in its own container. So it's purely for what I see as maintainability. Again, happy to remove it if you prefer.

Reply:
I had a look at the final design, and it turns out that this is necessary in order to implement the latest design.